### DISCUSSION TITLE
private readonly IOrchardHelper _orchardHelper;      var product = _orchardHelper.GetContentItemByIdAsync(productId);

### DISCUSSION THREAD

**Discussion by IdrissaMaiga** [OrchardCMS/OrchardCore#15742] [Q&A]
GetContentItemByIdAsync(productId);  
Method does not exist

**Comment by hishamco** [comment]
This is accessible via `IOrchardHelper`

https://github.com/OrchardCMS/OrchardCore/blob/b1b47175d438974813d040fbcca4f2e6764d47f2/src/OrchardCore.Modules/OrchardCore.Contents/Razor/ContentRazorHelperExtensions.cs#L52-L56

**Comment by IdrissaMaiga** [comment]
thank you very much

**Reply by hishamco** [reply]
Let me know if it's works wirth you
